### PR TITLE
Adds section for Serial2

### PR DESCRIPTION
**Pull request by mha1**
Adds section for Serial2

### CODE REVIEW

File: docs/software/serial-protocols.md [hunk 1]
```diff
@@ -59,6 +61,43 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
 ![output mapping serial](../assets/screenshots/receiver-serial.png)
 </figure>
 
+## Second Serial Interface on ESP32 PWM receivers
+
+Starting with firmware version 3.5 an additional serial interface (Serial2) can be assigned on any featured PWM Output to run any of the available serial protocols. Both serial interfaces can be be utilized at the same. This enables setups like running a CRSF wing stabilization unit and HoTT telemetry, Tramp or SmartAudio at the same time. Serial2 can also be utilized to simplify the wiring of e.g. an SBUS connection on receivers which have the first serial interface on separate connectors (e.g. RM receivers with JST connectors).
```
Comment (deadbytefpv) [inline docs/software/serial-protocols.md:66]: "can be utilized at the same time"? or use "simultaneously" as the next sentence is already using "same time".

File: docs/software/serial-protocols.md [hunk 2]
```diff
@@ -59,6 +61,43 @@ When connected to the receiver with the transmitter, enter the ExpressLRS Lua sc
...
+
+### Assigning Serial2 TX vs Serial2 RX and Serial2 TX
+
+Most of the supported serial protocols are uni-directional (only sending data) or are bi-directional (sending and receiving data) on one single wire (half duplex). For those protocols it is only necessary to assign a TX pin. True (full duplex) serial protocols require the assignment of RX and TX. The following list shows the configuration requirements for the supported protocols:
+
+CRSF and MAVLINK: require Serial2 TX and additionally Serial2 RX if the device returns data (e.g. telemetry)
```
Comment (deadbytefpv) [inline docs/software/serial-protocols.md:72]: format these as an unordered list.
i.e. add dash at the beginning of each line.

**Review by deadbytefpv — CHANGES_REQUESTED** [review]
I don't really like to be pedantic, but there are those who do, and I'd rather anticipate this before it gets annoying later on. ;)

File: docs/software/serial-protocols.md [hunk 3]
```diff
@@ -170,12 +209,13 @@ If an adapter cable is required depends on the platform the receiver is based on
 
 ESP32 based receivers with Serial on dedicated PWM servo connectors like the BetaFPV SuperP 14ch:
 
-- No adapter cable is required. HoTT sensors can be directly connected to the PWM servo connector dedicated for Serial TX.
+- No adapter cable is required. HoTT sensors can be directly connected to the PWM servo connector dedicated for Serial TX any of the PWM pins configured for Serial2 (see above section).
```
[the remark this answers is not in the document]
Comment (mha1) [reply docs/software/serial-protocols.md:212]: Thanks for reading this carefully. I don't call this pedantic, it's proof reading as it should be done.

I reworded the sentence,

File: docs/software/serial-protocols.md (hunk 1 above)
Comment (mha1) [reply docs/software/serial-protocols.md:66]: done

File: docs/software/serial-protocols.md (hunk 2 above)
Comment (mha1) [reply docs/software/serial-protocols.md:72]: done

**Review by deadbytefpv — APPROVED** [review]

**Pull request merged**